migration/src/m0000630_create_product_version_range.rs

```diff
@@ -0,0 +1,76 @@
+use sea_orm_migration::prelude::*;
+
+use crate::{m0000290_create_product::Product, UuidV4};
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
+                    .table(ProductVersionRange::Table)
+                    .if_not_exists()
```

Comment:
nit (non-blocking): I think we tried to stop using these conditionals like `if_not_exists()` because we want to know if we're tramping something or otherwise conflicting with a supposed known state of the system.